source/class/cv/io/rest/Client.js

```diff
@@ -39,10 +39,8 @@ qx.Class.define('cv.io.rest.Client', {
     getBaseUrl: function () {
       if (!this.BASE_URL) {
         let path = '';
-        const engine = cv.TemplateEngine.getInstance();
-        const clientBackend = engine.visu && typeof engine.visu.getBackend === 'function' ? engine.visu.getBackend() : {};
-        if (clientBackend.resources && clientBackend.resources.rest) {
-          path = clientBackend.resources.rest;
```

Comment:
Is this code path only relevant for OpenHAB?
Or could it also be used by KNX, e.g. when served by the Docker container?

Reply:
This has always been a mix-up of two different things. The URL of the CometVisus REST-API has nothing to do with the backend used for state handling. e.g. you can use openHAB as "state-backend" and still have the normal PHP-REST-API (when you use the docker container). Only if you serve the CometVisu via openHABs own webserver, you use the REST-API that the cometvisu binding provides (same features as the PHP stuff, but natively implemented in java to make it work in openhab). And fpr that you need a slightly different URL. And thats what this is all about. So you only need to know if the CometVisu is served via openHAB.